client/app/dbaas/logs/detail/streams/logs-streams.service.js

```diff
@@ -247,6 +251,73 @@ class LogsStreamsService {
         };
     }
 
+    /**
+     * extracts graylog URL from stream. Shows error message on UI if no graylog URL is found.
+     *
+     * @param {any} stream
+     * @returns {string} graylog url, if not found empty string
+     * @memberof LogsStreamsService
+     */
+    getStreamGraylogUrl (stream) {
+        const url = this.UrlHelper.findUrl(stream, "GRAYLOG_WEBUI");
+        if (!url) {
+            this.CloudMessage.error({ textHtml: this.$translate.instant("logs_streams_get_graylog_url_error", { stream: stream.info.title }) });
```

Comment:
I used this.ServiceHelper.errorHandler but it did not show the message on UI. On debugging I found that It only shows error message on UI if it is used inside catch() method. Because of this, I changed to this.CloudMessage.error.